src/main/scala/routemap.scala

```diff
@@ -10,22 +10,23 @@ object RouteMap {
   /**
    * Find all airports located in a particular city
    */
-  def findAirports(city: String): List[Airport] = OpenFlights.airports(city)
+  def findAirports(city: String): List[Airport] = {
+    if (OpenFlights.cityAirports.contains(city))
+      return OpenFlights.cityAirports(city)
+    else
+      return List[Airport]()
+  }
```

Comment:
This kind of change shouldn't be necessary if returning `OpenFlights.cityAirports` as a map `withDefaultValue` has been kept intact, but I haven't tried running any of this new code yet. At worst, you should be able to reduce it to `OpenFlights.cityAirports.getOrElse(List.empty[Airport])` if for some reason you couldn't do that.